lib/types/beacon_chain/execution_payload.ex

```diff
@@ -32,14 +32,14 @@ defmodule Types.ExecutionPayload do
           state_root: Types.root(),
           receipts_root: Types.root(),
           # size BYTES_PER_LOGS_BLOOM
-          logs_bloom: binary(),
+          logs_bloom: list(Types.uint8()),
           prev_randao: Types.bytes32(),
           block_number: Types.uint64(),
           gas_limit: Types.uint64(),
           gas_used: Types.uint64(),
           timestamp: Types.uint64(),
           # size MAX_EXTRA_DATA_BYTES
-          extra_data: binary(),
+          extra_data: list(Types.bytes1()),
```

Comment:
I did a test with a mainnet block and this part is not working it seems. I get

```
** (EXIT from #PID<0.98.0>) an exception was raised:
    ** (FunctionClauseError) no function clause matching in LambdaEthereumConsensus.SszEx.encode_fixed_size_list/3
        (lambda_ethereum_consensus 0.1.0) lib/ssz_ex.ex:342: LambdaEthereumConsensus.SszEx.encode_fixed_size_list("@penguinbuild.org", {:int, 8}, 32)
        (elixir 1.15.6) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
        (lambda_ethereum_consensus 0.1.0) lib/ssz_ex.ex:602: LambdaEthereumConsensus.SszEx.encode_schemas/1
        (lambda_ethereum_consensus 0.1.0) lib/ssz_ex.ex:571: LambdaEthereumConsensus.SszEx.encode_container/2
        (elixir 1.15.6) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
        (elixir 1.15.6) lib/enum.ex:1693: Enum."-map/2-lists^map/1-1-"/2
        (lambda_ethereum_consensus 0.1.0) lib/ssz_ex.ex:602: LambdaEthereumConsensus.SszEx.encode_schemas/1
        (lambda_ethereum_consensus 0.1.0) lib/ssz_ex.ex:571: LambdaEthereumConsensus.SszEx.encode_container/2
```

Weird this hasn't been caught by the spec test?

Reply:
Here is the branch, https://github.com/lambdaclass/lambda_ethereum_consensus/compare/ssz-ex-bench?expand=1

you can run it with `mix run --no-start bench/ssz_ex_bench.exs`

Reply:
@mpaulucci I pulled the latest changes to the new branch `ssz-ex-bench` and the bench works.

Reply:
The encoding is also slower :disappointed: